llvm/lib/Support/RISCVISAInfo.cpp

```diff
@@ -103,6 +103,8 @@ static const RISCVSupportedExtension SupportedExtensions[] = {
     {"zicbop", RISCVExtensionVersion{1, 0}},
 
     {"svnapot", RISCVExtensionVersion{1, 0}},
+
```

Comment:
Fixed in https://github.com/openhwgroup/corev-llvm-project/pull/36

Reply:
If this is a separate patch, perhaps clang's builtin support could also be a separate patch, so that no dependencies are needed and each patch is independent.

Reply:
Already merged